### ISSUE TITLE
Emacs as R-IDE using tramp (and, if possible, images)

### ISSUE THREAD

**Issue by n0542344** [eddelbuettel/t4#4]
tl;dr: I'd love an episode on 'how to transform Emacs into a comfy IDE for R' with a focus on using tramp (and, if possible, using inline-images, probably via org).

- - -  

I don't even know where to start, this T4-series is exactly what I was looking for! 

I'm using (neo)vim as my R-IDE for some time now and I'm quite content with it (RStudio is great, but my productivity increased enormously by switching away from it). But lately I found myself being drawn towards Emacs, primarily because of the prospect of using tramp-mode. But I have to admid that I wasn't able to transform Emacs into an IDE I was really comfortable to work with, although ESS is awesome (thanks for packaging that in Debian btw). 

In addition I'm unsure if it is possible to display images while working remote via tramp (or generally how to get inline-images to work) and I'd love an episode on that.

Best wishes, Alex (a life science PhD-student from Vienna)

**Comment by eddelbuettel** [comment]
Yes! Been using ESS since the 90s.  

Personally, I keep my R local to my session so I would rather ssh out and Emacs+ESS on the remote box so I won't go into remote ESS.  That also ... is going a little far.

But I should cover tramp. Good point, added to my notes.
